### ISSUE TITLE
Cannot reproduce errors on SPICE

### ISSUE THREAD

**Issue by AndChenCM** [yuyangw/Denoise-Pretrain-ML-Potential#1]
Dear authors,

Thanks for your nice work! I tried to reproduce the 'MAE on SPICE / TorchMD-Net' from **Figure 2 d** in your preprint using your config.yaml and your provided checkpoint with torchmdnet. The only two parameters I change are _batch_size_=64 and _num_workers_=16.

In your preprint, the MAE is **9.8** or **9.0**, without and with pretrain (50 epochs). But I got **17.26** and **18.50** in my two runs (10 epochs), which means the result got worse after loading the pretrained weights. What might be the problem? Besides, I wonder in your preprint, did you use a different config of torchmdnet? 

Best,
Ming-an

**Comment by yuyangw** [comment]
Hi Ming-an, 

Thank you for your interest in our work. Could you please share the full training details?

Best, 
Yuyang

**Comment by AndChenCM** [comment]
Sorry, I previously used only 10 epochs and batch_size 64 for training, which might make the numbers meaningless. Now, I tried to run 50 epochs and got MAE 13 or 10 w/o or w/ pretrain, using the same config (i.e., same batch_size and torchmdnet config) as you provided in github. My question now is have you ever run the training until convergence? What is the best possible MAE you can get on SPICE?

**Comment by yuyangw** [comment]
As we reported in the manuscript. We train models for 50 epochs on SPICE. Training for more epochs with other learning rate scheduler settings could achieve lower MAE/RMSE. However, we didn't try that in our current work. Hope this helps.

Best, 
Yuyang
